docs/agentd/agent_webhook.md

```diff
@@ -0,0 +1,18 @@
+# Webhook Usage in Aries-agentd
+
+Aries-agentd uses a webhook mechanism to communicate events back to the controller.
+
+The URL that aries-agentd should send events to can be set with the `--webhook-url` command line argument or with the `ARIESD_WEBHOOK_URL` environment variable.
+
+Currently, aries-agentd will notify the controller of any connection notifications.
```

Comment:
"event notifications"? webhook is not only for connections.

Reply:
By that I just meant what's supported in the code currently. But I suppose that's obvious - webhook only supports what features we have in the code... I'll make the change.

Reply:
Updated